src/bindings/python/tests/test_graph/test_convolution.py

```diff
@@ -5,143 +5,66 @@
 import numpy as np
 
 import openvino.runtime.opset8 as ov
-from tests.runtime import get_runtime
-from tests.test_graph.test_ops import convolution2d
-from tests.test_graph.util import run_op_node
 
 
 def test_convolution_2d():
```

Comment:
Are you able to parametrize this kind of tests? Or at least split test cases?

I see a lot repetitions for the same operator. Also there is a lot of redefinitions like `strides`/`pads_begin` etc.

Reply:
I'll get to it, let's move the discussion to the second part.